docs/source/getting-started/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _getting-started:
+
+Getting Started
+===============
+
+This page will show how to start using MythX in the shortest number of steps.
+
+Quickstart
+----------
+
+The quickest way to see MythX in action is by using a MythX tool such as the :ref:`MythX for Truffle <tooling.truffle>` plugin.
+
+No registration is required for this, though the results will be limited. See the section on :ref:`using MythX for Truffle <tooling.truffle>` for more details.
+
+#. In a terminal, navigate to the root of your local Truffle project.
+
+#. Install the MythX plugin:
+
+     .. code-block:: console
+
+        npm install truffle-security
+
+#. Analyze all the contracts in the project with MythX:
+
+     .. code-block:: console
+
+        truffle run verify
+
+This will run a security analysis in Trial Mode, which limits the returned results to three vulnerabilities only. For a more complete report, you will need to create an account with MythX.
+
+
+Creating an account
+-------------------
+
+To get the full functionality of MythX, you need to create an account.
+
+.. note:: MythX requires `MetaMask <https://metamask.io>`_, a browser-based wallet application, in order to create accounts. You must be logged in to MetaMask before continuing. The active address will be associated with your account and be used for any plans you may purchase.
+
+#. Go to https://mythx.io and slick the :guilabel:`Sign Up` button on the top right.
+
+#. Fill out the registration form. You will need to supply a valid email address.
+
+   .. image:: img/registration.png
+
+#. When finished, click :guilabel:`Complete Registration`.
+
+#. A MetaMask popup will display, asking for confirmation. Click :guilabel:`Sign` to continue.
+
+   .. image:: img/metamasksignup.png
+      :width: 50%
+
+#. An API key will be generated for you and displayed. Please copy this key down, as you will not be able to retrieve it later. (You can generate a new one in your account dashboard if necessary.)
```

Comment:
I had a call recently with Sharon and it seems like API Key will be renamed to Password (which is the name currently in production).
 My reasoning to use API key rather than Password is that it's being automatically generated by us, not by the user. However:
- All the API documentation and parameter names (for instance we need to pass a `password` property on the API's http body of the authentication request) use the term `password`;
- Same goes for the tools which read the secret from an environment variable (`export MYTHX_PASSWORD='Put your password in here!'`)

So it seems better to just keep calling it password. What do you think?
cc @b-mueller

Reply:
Hmm, actually I could see it either way. The fact that the environment variables have thus far been called PASSWORD makes me lean to password and not API key, but I could be convinced otherwise if someone feels stronger about it.

Reply:
Would a user be able to change their own password, or would it still need to be "regenerated"? I ask only because the latter doesn't feel like what a user might expect for a password. Maybe we auto generate the first one, and let the user change it subsequently?

Reply:
Currently the dashboard is only exposing the functionality to generate new passwords, but this is all done on the client side, meaning an advanced user can just call our API directly and set any password he wants (as long as it obeys the password rules, x characters, y numbers, z special characters, etc)